pkg/cmd/cli/describe/describer.go

```diff
@@ -336,3 +341,70 @@ func (d *PolicyBindingDescriber) Describe(namespace, name string) (string, error
 		return nil
 	})
 }
+
+// TemplateDescriber generates information about a template
+type TemplateDescriber struct {
```

Comment:
can we start including the describer impls with the packages they describe instead of putting them all in this file?  
1) leads to cleaner commits/less conflicts when everyone starts needing to tweak this file
2) makes it more obvious when you add a field to an object, that you need to update the describe method (because it's right there in the package, and ideally the file, you're already working in)